### ISSUE TITLE
Stop supporting kubeflow/mxjob integration

### ISSUE THREAD

**Issue by tenzen-y** [kubernetes-sigs/kueue#1429]
**What would you like to be cleaned**:
We should stop supporting the kubeflow/mxjob integration.

**Why is this needed**:
Due to Apache MXNet having been archived, the kubeflow community plans to remove MXJob from the training-operator.

https://github.com/kubeflow/training-operator/pull/1953

**Comment by tenzen-y** [comment]
We need to watch #1953, and then we can decide when we stop supporting this integration.

**Comment by terrytangyuan** [comment]
Yes, I don't have any concerns about dropping Apache MXNet support.

**Comment by tenzen-y** [comment]
/lifecycle frozen.

Kubeflow Community decided to remove MXJob after 1 minor release.
So, we can remove the kubefow/mxjob integration after it.

**Comment by kannon92** [comment]
@tenzen-y 

With https://github.com/kubernetes-sigs/kueue/pull/4077, is there any reason to keep this doc open?

**Comment by mimowo** [comment]
>  doc open?

i assume "issue" open, I don't think there is, closing
/close
